core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/waitfor/WaitForHelper.java

```diff
@@ -0,0 +1,27 @@
+package com.cognifide.aet.job.common.modifiers.waitfor;
+
+import com.cognifide.aet.communication.api.metadata.CollectorStepResult;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+
+public class WaitForHelper {
+
+    public static CollectorStepResult waitForExpectedCondition(
```

Comment:
Could you please provide short javadoc for this method? I believe it can be reused in the future by other modifiers.